### DISCUSSION TITLE
[15일차 -윤경] Box model에 대해 설명해주세요.

### DISCUSSION THREAD

**Discussion by skyhanull** [zzangCS/zzangInterview#52] [프론트 최신 이슈]
박스모델에 대해 설명해주세요

**Comment by ez1n** [comment]
CSS의 Box Model은 HTML element가 웹 페이지에서 차지하는 공간을 정의한 모델입니다.
HTML element들은 각각 자신만의 영역을 갖고 있으며 각 영역은 다시 여러개의 작은 영역으로 나뉩니다.

각 element는 가운데 실제 내용이 담긴 content, 경계인 border, border과 content 사이의 영역인 padding, border 바깥 영역인 margin으로 구성됩니다.
